bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerDescriptorImpl.java

```diff
@@ -122,7 +123,7 @@ ByteBuf getExplicitLac() {
             result = logFenceResult = SettableFuture.create();
         }
         ByteBuf entry = createLedgerFenceEntry(ledgerId);
-        journal.logAddEntry(entry, (rc, ledgerId, entryId, addr, ctx) -> {
+        journal.logAddEntry(entry, LedgerType.FORCE_ON_JOURNAL, (rc, ledgerId, entryId, addr, ctx) -> {
```

Comment:
Just repeating my comment. We must not overload ledger type to enforce immediate flush/persist. If needed we need to add another argument.

Reply:
done